index.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
-if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./formik.cjs.production.js');
-} else {
+if (__DEV__) {
```

Comment:
Will this file appear in node_modules as is?

Reply:
/cc @jaredpalmer

Reply:
it is replaced by `process.env.NODE_ENV === 'production'`

But as you can see, it is too verbose to type in every time

Reply:
Ah, ok.